docs/quick-start/starknet-devnet-with-asdf.mdx

```diff
@@ -0,0 +1,54 @@
+# StarkNet Devnet - Installation and Setup with asdf
+
+This document provides a guide to install and run **StarkNet Devnet** using `asdf`, a Rust-based tool for developers. Below, you'll find instructions for installation using `asdf`.
+
+## Requirements
+
+- **asdf**: Install `asdf` by following the [official asdf documentation](https://asdf-vm.com/).
+
+## Installation Using `asdf`
+
+1. **Add the StarkNet Devnet plugin**:
+
+   ```bash
+   asdf plugin add starknet-devnet
+   ```
+
+2. **List all available versions**:
+
+   ```bash
+   asdf list-all starknet-devnet
+   ```
+
+3. **Install the latest version**:
+
+   ```bash
+   asdf install starknet-devnet latest
+   ```
+
+4. **Set the installed version as global or local**:
+   ```bash
+   asdf global starknet-devnet <VERSION>
+   # or
+   asdf local starknet-devnet <VERSION>
+   ```
+
```

Comment:
You should also add an example with the version used, or indicate the correct version that we are using, in this case "0.2.3", we can refer to reviewing the current supported version [Starknet-Devnet in README of Scaffold](https://github.com/Scaffold-Stark/scaffold-stark-2?tab=readme-ov-file#starknet-devnet-version), since sometimes it is not updated at the same time.